### PR TITLE
Enable custom classloader

### PR DESCRIPTION
**Pull request by molly-moen**
Enable the custom class loader in javabuilder. Once this is deployed, users will see the following error messages if they use a disallowed class:

## Regular Code Execution
Screen Shot 2022-01-05 at 10 55 36 AM

## Running a test
Screen Shot 2022-01-05 at 10 55 49 AM

Note: this PR depends on [this PR](https://github.com/code-dot-org/javabuilder/pull/177) being merged first.

### CODE REVIEW

File: org-code-javabuilder/lib/src/main/java/org/code/javabuilder/UserClassLoader.java [hunk 1]
```diff
@@ -46,16 +46,13 @@ public UserClassLoader(URL[] urls, ClassLoader parent, List<String> userProvided
       }
     }
 
-    // Log that we are going to throw an exception
+    // Log that we are going to throw an exception. Log as a warning
+    // as it is most likely user error, but we want to track it.
     JSONObject eventData = new JSONObject();
     eventData.put("type", "invalidClass");
```
Comment (sanchitmalhotra126) [inline org-code-javabuilder/lib/src/main/java/org/code/javabuilder/UserClassLoader.java:52]: nit: `LoggerConstants.TYPE`?

File: org-code-javabuilder/lib/src/main/java/org/code/javabuilder/UserClassLoader.java [hunk 2]
```diff
@@ -46,16 +46,13 @@ public UserClassLoader(URL[] urls, ClassLoader parent, List<String> userProvided
...
     eventData.put("className", name);
```
Comment (sanchitmalhotra126) [inline org-code-javabuilder/lib/src/main/java/org/code/javabuilder/UserClassLoader.java:53]: LoggerConstant for "className" as well?

**Review by sanchitmalhotra126 — APPROVED** [review]
Small nits, otherwise LGTM!

File: org-code-javabuilder/lib/src/main/java/org/code/javabuilder/UserClassLoader.java (hunk 1 above)
Comment (molly-moen) [reply org-code-javabuilder/lib/src/main/java/org/code/javabuilder/UserClassLoader.java]: done!

File: org-code-javabuilder/lib/src/main/java/org/code/javabuilder/UserClassLoader.java (hunk 2 above)
Comment (molly-moen) [reply org-code-javabuilder/lib/src/main/java/org/code/javabuilder/UserClassLoader.java]: done!

**Pull request merged**